### DISCUSSION TITLE
Functionality to find where baselined errors are

### DISCUSSION THREAD

**Discussion by Zeckie** [KotlinIsland/basedmypy#117] [Ideas]
**Feature**

Need some way to easily see where the baselined errors are (eg. at a module level). Would be great if it could also be able to compare 2 baselines to show which files / modules have new vs resolved errors.

**Pitch**

Would be useful to be able to easily see which modules have baselined errors, and what errors are being introduced / removed from the baseline in a change (Merge / Pull request).

**Comment by DetachHead** [comment]
outputting baselined errors as notes seems like a decent solution imo

**Comment by KotlinIsland** [comment]
#93 and #111 will resolve viewing the baseline.

#116 will address diffing two baselines.

**Reply by KotlinIsland** [reply]
#185 is also related. Perhaps a `--baseline-summary` command would be good.
> How many different types of error (in the baseline / code)

I really like this idea.
